### PR TITLE
feat(accordion): add accordion component (part1)

### PR DESCRIPTION
**Pull request by nadiia**
screen shot 2018-11-06 at 11 31 30 am

### CODE REVIEW

File: src/accordion/accordion.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    this.setState(newState);
+    onChange(newState);
+  }
+
+  getItems() {
+    const {expanded} = this.state;
+    const {accordion, disabled, children} = this.props;
+    const newChildren = [];
+
+    React.Children.forEach(children, (child, index) => {
```
Comment (gergelyke) [inline src/accordion/accordion.js]: can we use `React.Children.map` here so we can get rid of the side effect of the iterator function?

**Comment by gergelyke** [comment]
should we add the `aria-controls` attribute to the implementation? http://web-accessibility.carnegiemuseums.org/code/accordions/

File: src/accordion/accordion.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+// @flow
```
Comment (austin94) [inline src/accordion/accordion.js]: [nit] Does this file need the copyright header?

Comment (nadiia) [reply src/accordion/accordion.js]: it does! let me fix it

**Comment by nadiia** [comment]
> should we add the `aria-controls` attribute to the implementation? http://web-accessibility.carnegiemuseums.org/code/accordions/

Created an issue for myself to add it in a follow-up PR #473

**Pull request merged**